Server/Controllers/MatchController.cs

```diff
@@ -372,4 +498,15 @@ public class MatchEventsRequest
     {
         [Required] public long MatchId { get; set; }
     }
+
+    public class AfterMatchResponse
+    {
+        public IEnumerable<LeaderBoardPlayer> LeaderBoard { get; set; }
```

Comment:
When match ends we don't need fresh leaderboards, we need only old and new rating for players in the match

```suggestion
        public IEnumerable<Player> Players { get; set; }
        public class Player
        {
            public string SteamId { get; set; }
            public int OldRating { get; set; }
            public int NewRating { get; set; }
        }
```